src/vstart.sh

```diff
@@ -1162,7 +1162,22 @@ EOF
 	fi
         ceph_adm config-key set mgr/cephadm/ssh_identity_key -i ~/.ssh/id_rsa
         ceph_adm config-key set mgr/cephadm/ssh_identity_pub -i ~/.ssh/id_rsa.pub
-        ceph_adm mgr module enable cephadm
+        local retry_flag=0
```

Comment:
It's a counter, not a flag, so I'd suggesting renaming to `num_retries` or `retry_counter`.

Reply:
Modified too.